azure/flatcar-linux/kubernetes/bootkube.tf

```diff
@@ -1,6 +1,6 @@
 # Self-hosted Kubernetes assets (kubeconfig, manifests)
 module "bootkube" {
-  source = "git::https://github.com/poseidon/terraform-render-bootkube.git?ref=b96d641f6d42cf5d9bf3ac36f557aa21cc157680"
+  source = "git::https://github.com/kinvolk/terraform-render-bootkube.git?ref=d07243a9e7f6084cfe08b708731a79c26146badb"
```

Comment:
This is an older commit https://github.com/kinvolk/terraform-render-bootkube/commit/d07243a9e7f6084cfe08b708731a79c26146badb. Newer commit hash is `365b801d692804a5bbd9bb96ffff3531acb3c8d1` https://github.com/kinvolk/terraform-render-bootkube/commit/365b801d692804a5bbd9bb96ffff3531acb3c8d1